style.css

```diff
@@ -148,10 +148,71 @@ body {
   color: #fff;
   border-radius: 4px;
   background: #0056EC;
+  border: 0;
+  transition: background 0.2s ease-out 0s;
+  outline: none;
 }
 
 .main__button:hover {
   cursor: pointer;
   background: #1e90ff;
-  transition: 1s;
 }
+
+.footer {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  max-width: 80%;
+}
+.footer__circle {
+  position: relative;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  opacity: .7;
+  margin-right: 4px;
+  background: #c8dbfb;
+}
+
+.footer__circle:last-child {
+  margin-right: 0;
+}
+
+@media (max-width: 1366px) {
+  .wrapper {
+  max-width: 68%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tni-htmlcss2.0/pullRequest?prid=6206132)